include/quic/gnutls_crypto.hpp

```diff
@@ -25,7 +25,9 @@ namespace oxen::quic
             unsigned int incoming,
             const gnutls_datum_t* msg)>;
 
-    constexpr size_t GNUTLS_KEY_SIZE = 32;  // for now, only supporting Ed25519 keys (32 bytes)
+    inline constexpr size_t GNUTLS_KEY_SIZE = 32;  // for now, only supporting Ed25519 keys (32 bytes)
```

Comment:
Suggested rename `GNUTLS_PUBKEY_SIZE` or `GNUTLS_PUBLIC_KEY_SIZE`.  Just `key` is ambiguous when you're in public/private keypair land, which we are with Ed25519 keys.